girder/stemserver_plugin/models/stemimage.py

```diff
@@ -1,24 +1,30 @@
-import os
-
 import h5py
 import numpy as np
 
 from girder.api.rest import setResponseHeader, RestException
-from girder.constants import AccessType
+from girder.constants import AccessType, AssetstoreType
+from girder.models.assetstore import Assetstore as AssetstoreModel
 from girder.models.file import File as FileModel
+from girder.models.folder import Folder as FolderModel
+from girder.models.item import Item as ItemModel
 from girder.models.model_base import AccessControlledModel
+from girder.utility.filesystem_assetstore_adapter import (
+    FilesystemAssetstoreAdapter
+)
 
 
 class StemImage(AccessControlledModel):
 
+    IMPORT_FOLDER = 'stem_images'
+    IMPORT_ITEM_NAME = '_imported_images'
```

Comment:
Was there a particular reason for just having a single item, rather than the standard Girder thing of have a item with the same name as the file? This approach saves space I guess.

Reply:
Well, when we call [importFile](https://github.com/girder/girder/blob/292690e7e4c269ed3b34757ba86ddfa2713f9f16/girder/utility/filesystem_assetstore_adapter.py#L319), we have to pass it an already existing item where the file will be placed. 

I thought it would save time to just have one already created item that we put all the files in, rather than having to create a new item every time before we import a new file.

Reply:
Fair enough